functions/Get-YouTubeChannel.ps1

```diff
@@ -22,6 +23,10 @@ function Get-YouTubeChannel {
   $Result = Invoke-RestMethod -Uri $Uri -Method Get -Headers (Get-AccessToken)
   $Result.items | ForEach-Object -Process { $PSItem.PSTypeNames.Add('YouTube.Channel') }
 
-  if ($Raw) { return $Result }
-  $Result.items
+  if ($Raw) { 
```

Comment:
Why did you change the logic here? It's more concise to simply return `$Result` in an `if` statement. If that parameter isn't passed in, then skip and return the items. No need to put an else statement.